reference/forms/types/dateinterval.rst

```diff
@@ -0,0 +1,290 @@
+.. index::
+   single: Forms; Fields; DateIntervalType
+
+DateIntervalType Field Type
+===========================
+
+This field type allows the user to modify data that represents a specific
+date interval. It can be rendered as a integer or text input or select tags.
+
+The underlying format of the data can be a ``DateInterval`` object,
+a `RFC 3339`_ duration string (e.g. ``P1DT12H``) or an array.
```

Comment:
Are we sure we want to link to the RFC 3339? In that RFC you can read the following:

![rfc_3339](https://cloud.githubusercontent.com/assets/73419/16553141/ba340846-41c6-11e6-8876-0af1e8c02b6c.png)

It looks like we should link instead to ISO 8601 standard: https://en.wikipedia.org/wiki/ISO_8601